src/BSD-3-Clause.xml

```diff
@@ -5,14 +5,15 @@
       <crossRefs>
          <crossRef>https://opensource.org/licenses/BSD-3-Clause</crossRef>
          <crossRef>https://www.eclipse.org/org/documents/edl-v10.php</crossRef>
+         <crossRef>https://github.com/sphinx-doc/alabaster/blob/0.x/LICENSE</crossRef>
```

Comment:
I'm probably inclined not to add a crossRef for sphinx-doc specifically, since we haven't for the many other instances of BSD-3-Clause markup variations out there in the wild.